tests/components/blueprint/snapshots/test_importer.ambr

```diff
@@ -1,6 +1,6 @@
 # serializer version: 1
 # name: test_extract_blueprint_from_community_topic
-  NodeDictClass({
+  dict({
```

Comment:
I believe what is happening here is the inputs used to come from the yaml loader, so they were the special class `NodeDictClass`, but now the inputs are put together as a dictionary manually in the `inputs` function, as it has to merge them together from several sections, so that's why the snapshot need to be updated.